class ErrorMessage(Exception):
    """Собственное исключение."""

    pass